### DISCUSSION TITLE
v6.2.2

### DISCUSSION THREAD

**Discussion by fcakyon** [fcakyon/yolov5-pip#163] [Announcements]
## What's Changed
* update workflow versions by @fcakyon in https://github.com/fcakyon/yolov5-pip/pull/156
* fix a bug in train resume by @QazyBi in https://github.com/fcakyon/yolov5-pip/pull/161

## New Contributors
* @QazyBi made their first contribution in https://github.com/fcakyon/yolov5-pip/pull/161

**Full Changelog**: https://github.com/fcakyon/yolov5-pip/compare/6.2.1...6.2.2

<hr /><em>This discussion was created from the release <a href='https://github.com/fcakyon/yolov5-pip/releases/tag/6.2.2'>v6.2.2</a>.</em>

**Comment by justlike-prog** [comment]
Hi @fcakyon . It seems that if I import yolov5 and opencv (version 4.6.0). the imwrite operation doesn't work correctly.
Reproduce by:

```
import yolov5
import cv2

import cv2 as cv
test = cv.imread("img1.png")

cv.imwrite("img1_test.png", test, [cv.IMWRITE_JPEG_QUALITY,75])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: imwrite() takes 2 positional arguments but 3 were given
```

while this works fine when not importing yolov5.

**Reply by fcakyon** [reply]
Can you paste the full traceback, this doesnt seem to be a yolov5-pip related issue.

Can you try as 
import cv2
cv2.imread
